src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2154,9 +2154,22 @@ private ISwitchExpressionOperation CreateBoundSwitchExpressionOperation(BoundSwi
         {
             IOperation value = Create(boundSwitchExpression.Expression);
             ImmutableArray<ISwitchExpressionArmOperation> arms = CreateFromArray<BoundSwitchExpressionArm, ISwitchExpressionArmOperation>(boundSwitchExpression.SwitchArms);
+
+            bool isExhaustive = !boundSwitchExpression.ReportedNotExhaustive;
```

Comment:
>!boundSwitchExpression.ReportedNotExhaustive [](http://example.com/codeflow?start=32&length=44)

Taking a quick look at the lowering code, it appears that we can simply check if ```DefaultLabel``` is null for the node.